### PR TITLE
Adds some new settings for conferences and applauses

### PR DESCRIPTION
**Pull request by GabrielInTheWorld**
Please give me a feedback if the changes are sufficient.

### CODE REVIEW

File: openslides_backend/models/models.py [hunk 1]
```diff
@@ -253,6 +253,17 @@ class Meeting(Model):
     conference_open_microphone = fields.BooleanField(default=False)
     conference_open_video = fields.BooleanField(default=False)
     conference_auto_connect_next_speakers = fields.IntegerField(default=0)
+    conference_enable_helpdesk = fields.BooleanField(default=False)
```
Comment (jsangmeister) [inline openslides_backend/models/models.py:256]: This file is autogenerated, as stated at the start. Please add these changes to the [`models.yml`](https://github.com/OpenSlides/OpenSlides/blob/openslides4-dev/docs/models.yml) and execute `make generate-models` after they are merged.

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: openslides_backend/action/actions/meeting/update.py [hunk 2]
```diff
@@ -32,6 +32,17 @@
     "conference_los_restriction",
     "conference_stream_url",
     "conference_stream_poster_url",
+    "conference_open_microphone",
+    "conference_open_video",
+    "conference_auto_connect_next_speakers",
+    "conference_enable_helpdesk",
+    "applause_enable",
```
Comment (FinnStutzenstein) [inline openslides_backend/action/actions/meeting/update.py:39]: All these fields are not in the models.yml! Do not add them here until you changed the models.

Comment (GabrielInTheWorld) [reply openslides_backend/action/actions/meeting/update.py:39]: See the comment above from @jsangmeister

File: openslides_backend/models/models.py (hunk 1 above)
Comment (GabrielInTheWorld) [reply openslides_backend/models/models.py:257]: See OpenSlides/OpenSlides#6153

**Comment by jsangmeister** [comment]
Tests are failing because of other changes to the models file. You can work around that by manually removing the fields and then changing the checksum at the top in some way to indicate that it is not up-to-date. The new changes can be re-generated in #818.

**Comment by jsangmeister** [comment]
https://github.com/OpenSlides/openslides-backend/pull/840/ fixed all issues. You should be able to rebase and merge this (only changes in `meeting.update` should stay)

**Comment by jsangmeister** [comment]
Rebased by myself

**Review by jsangmeister — APPROVED** [review]

**Comment by GabrielInTheWorld** [comment]
> Rebased by myself

Okay. And what about the checksum-test?

**Comment by jsangmeister** [comment]
> > Rebased by myself
> 
> Okay. And what about the checksum-test?

That is okay to fail, it just means there are some changes to the models.yml which we did not integrate into the backend yet.

**Comment by GabrielInTheWorld** [comment]
> > > Rebased by myself
> > 
> > 
> > Okay. And what about the checksum-test?
> 
> That is okay to fail, it just means there are some changes to the models.yml which we did not integrate into the backend yet.

Okay, then I will merge it.

**Pull request merged**